src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,21 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private List<Bulldozer> list;
```

Comment:
I dont think we need this List as class level parameter, lets make it local variable (same in other producers)